go/libkb/globals.go

```diff
@@ -176,9 +175,8 @@ func (g *GlobalContext) Init() *GlobalContext {
 	g.createLoginState()
 	g.Resolver = NewResolver(g)
 	g.RateLimits = NewRateLimits(g)
-	g.CachedUserLoader = NewCachedUserLoader(g, CachedUserTimeout)
-	g.UserDeviceCache = NewUserDeviceCache(g)
-	g.makeFullSelfCacher()
+	g.upakLoader = NewUncachedUPAKLoader(g)
+	g.fullSelfer = NewNullFullSelfCache(g)
```

Comment:
the idea is that it won't crash, but it won't cache if you were to run it after the global context object initializes.  Later, when we {{ConfigureCaches}}, we get a real implementation swapped in.  I agree the name is bad, should I rename it?  `NewUncachedFullSelfer` ?